R/pkg/R/mllib.R

```diff
@@ -632,3 +642,159 @@ setMethod("predict", signature(object = "AFTSurvivalRegressionModel"),
           function(object, newData) {
             return(dataFrame(callJMethod(object@jobj, "transform", newData@sdf)))
           })
+
+
+#' Alternating Least Squares (ALS) for Collaborative Filtering
+#'
+#' \code{spark.als} learns latent factors in collaborative filtering via alternating least
+#' squares. Users can call \code{summary} to obtain fitted latent factors, \code{predict}
+#' to make predictions on new data, and \code{write.ml}/\code{read.ml} to save/load fitted models.
+#'
+#' For more details, see
+#' \href{http://spark.apache.org/docs/latest/ml-collaborative-filtering.html}{MLlib:
+#' Collaborative Filtering}.
+#' Additional arguments can be passed to the methods.
+#' \describe{
+#'    \item{nonnegative}{logical value indicating whether to apply nonnegativity constraints.
+#'                       Default: FALSE}
+#'    \item{implicitPrefs}{logical value indicating whether to use implicit preference.
+#'                         Default: FALSE}
+#'    \item{alpha}{alpha parameter in the implicit preference formulation (>= 0). Default: 1.0}
+#'    \item{seed}{integer seed for random number generation. Default: 0}
+#'    \item{numUserBlocks}{number of user blocks used to parallelize computation (> 0).
+#'                         Default: 10}
+#'    \item{numItemBlocks}{number of item blocks used to parallelize computation (> 0).
+#'                         Default: 10}
+#'    \item{checkpointInterval}{number of checkpoint intervals (>= 1) or disable checkpoint (-1).
+#'                              Default: 10}
```

Comment:
I see.  I think it's ok though. R ALS has 16 parameters - https://cran.r-project.org/web/packages/ALS/ALS.pdf

Since most parameters have default values people would just omit them. I think it is better to be explicit because otherwise it is easier to make mistakes on types (eg. c() vs list()), typos and so on.